### PR TITLE
[SQL Server] Fix region issue on mysql and sql server.

### PR DESCRIPTION
**Pull request by shenqianjin**

### CODE REVIEW

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/mysql/MySQLMvpModel.java [hunk 1]
```diff
@@ -73,9 +78,17 @@ public static Server findServer(final String subscriptionId, final String resour
     public static Server create(final String subscriptionId, final String resourceGroupName, final String serverName,
                                 Region region, final ServerPropertiesForDefaultCreate properties) {
         final MySQLManager manager = AuthMethodManager.getInstance().getMySQLManager(subscriptionId);
-        /*Sku sku = new Sku().withName("GP_Gen5_4");*/
+        List<PerformanceTierPropertiesInner> tiers = manager.locationBasedPerformanceTiers().inner().list(region.getName());
+        PerformanceTierPropertiesInner tier = tiers.stream().filter(e -> StringUtils.equals("Basic", e.id())).findFirst().orElse(
+            tiers.stream().filter(e -> StringUtils.equals("GeneralPurpose", e.id())).findFirst().orElse(
```
Comment (andxu) [inline Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/mysql/MySQLMvpModel.java:83]: can merge in one predicate function

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/database/RegionComboBox.java [hunk 2]
```diff
@@ -65,10 +65,12 @@ protected String getItemText(Object item) {
     }
 
     @Override
-    public void setItem(Region item) {
-        super.setItem(item);
-        if (!Objects.equals(item, getItem())) {
+    public void setSelectedItem(Object value) {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/database/RegionComboBox.java:68]: cannot understand the reason of the changes in this method

Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/database/RegionComboBox.java:68]: if value is changed, update the "changed" property.

Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/database/RegionComboBox.java:68]: avoid to validate when the region is not changed?

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/mysql/MySQLMvpModel.java (hunk 1 above)
Comment (shenqianjin) [reply Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/mysql/MySQLMvpModel.java:83]: no. the sort of these predication is considered by the code.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/database/RegionComboBox.java (hunk 2 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/database/RegionComboBox.java:68]: yes

**Review by andxu — DISMISSED** [review]

**Review by andxu — APPROVED** [review]

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/mysql/MySQLMvpModel.java (hunk 1 above)
Comment (andxu) [reply Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/mysql/MySQLMvpModel.java:83]: refactor

**Review by andxu — APPROVED** [review]

**Pull request merged**